applications/NFT_Bridge_Protocol_for_NFT_Migration_and_Data_Exchange.md

```diff
@@ -28,133 +28,449 @@ In this proposal, we use the following terms and definitions:
 * **Work Days:** 21 days
 * **Cost per work day:** £300
-* **Costs:** £6,300 (8558.05 USD/DAI)
-* **Deliverable:** A list of clearly defined, measurable features that the final bridges built using this vision should support, as well as how to test them using pseudocode.    
-
-We will write up and engage with the community to determine the desirable features of a NFT migration and data exchange messaging protocol.
-So far, we have determined that the protocol must allow for the following features:
-* Migrate any ERC-721 token between EVM-compatible chains
-* Migrate an NFT back and forth to get the original token back.
-* Be ERC-1155 compatible
-* Migrate an NFT outside of an EVM universe and redeem the same token back
-* Migrate an NFT to an arbitrary world (as the owner of an NFT, you should be able to choose where you want the NFT to be migrated to and who will mint it i.e. the protocol is not restricted to a specific project)
-* Have built-in protections against “bad” migration by preventing migration to non-registered destination worlds.
-* Allowing polkadot parachains to trustlessly migrate NFTs from one parachain to an another using SPREE.
-* Perform both IOU migration (no Digital Rights Management requirement) and Full migration (which requires Digital Rights Management at the bridge level)
-* Migrate an NFT with either a ‘checked’ migration (need to be acknowledged by both the sender and the receiver of a token) or ‘unchecked’ migration (the token in the original world needs to first be attributed to a tokenID on the destination chain, then it is simply put in escrow with the bridge on the origin chain). NB: Checked migrations are fully trutless and decentralized and optimised for security, whilst unchecked migrations are optimised for gas costs. 
-* Licensed under the [Unlicense](https://unlicense.org)            
-
-### Milestone 2 — Checked Migration Process
-* **Estimated Duration:** ~2 weeks
-* **FTE:** 2
-* **Work Days:** 20 days
+* **Costs:** £6,300 (8464.24 USD/DAI)
+* **Deliverable:** A list of features that the final bridges built using this vision should
+support, as well as how to test them.
+
+Previously we engaged with the community on numerous channels over the course of two to
+three months to determine the most desirable features of an NFT migration and data
+exchange messaging protocol. These features were included in accordance with both the
+spirit of our endeavour, and the communities wishes in mind.
+
+ See [A Protocol for NFT Migration](https://docs.google.com/document/d/1c5Uor2By5igFWXimipcKhsWjTAG8OWrl9bSVWTPsi6U/edit#heading=h.rflpba7y4sfj) for a more detailed explanation.
+
+The table below is actually the Deliverable itself along [this document](https://docs.google.com/document/d/1c5Uor2By5igFWXimipcKhsWjTAG8OWrl9bSVWTPsi6U/edit#heading=h.rflpba7y4sfj), as it was preparatory work into what the bridge would need to be able to do. Their implementation is the deliverable of the next milestones.
+
+
+<table>
```

Comment:
Would be nice to have this as a table. I'm sure you would find an automatic converter but of course it's not required.

Reply:
Done

Reply:
Again, not a requirement, but I meant as markdown tables.

Reply:
Done